app/src/theme/index.tsx

```diff
@@ -524,3 +524,27 @@ export function ThemeProvider({ children }: { children: React.ReactNode }) {
 
   return <StyledThemeProvider theme={themeAggregator(darkMode)}>{children}</StyledThemeProvider>
 }
+
+const TextWrapper = styled.div<{ color: keyof Colors }>`
+  color: ${({ color, theme }) => (theme as any)[color]};
+  font-family: Roboto;
+  letter-spacing: 0.2px;
+`
+
+export const TYPE = {
+  heading1(props: any) {
+    return <TextWrapper fontSize={'22px'} fontWeight={500} letterSpacing={'0.8px'} lineHeight={'26px'} {...props} />
+  },
+  heading2(props: any) {
+    return <TextWrapper fontSize={'18px'} fontWeight={400} lineHeight={'21px'} {...props} />
+  },
+  heading3(props: any) {
+    return <TextWrapper fontSize={'16px'} fontWeight={500} lineHeight={'19px'} {...props} />
+  },
+  bodyMedium(props: any) {
+    return <TextWrapper fontSize={'14px'} fontWeight={500} lineHeight={'16px'} {...props} />
+  },
+  bodyRegular(props: any) {
+    return <TextWrapper fontSize={'14px'} fontWeight={400} lineHeight={'16px'} {...props} />
```

Comment:
@Frankaus bodyMedium and bodyRegular seems to have line-height of 18px
![Screenshot 2021-07-06 at 21 14 29](https://user-images.githubusercontent.com/33226956/124654788-485aab00-de9f-11eb-88ea-2861e908d818.png)

Reply:
Good catch thank you!